Source/WebCore/css/parser/CSSPropertyParserConsumer+MetaConsumerDefinitions.h

```diff
@@ -119,6 +119,20 @@ template<typename Raw> bool isValidCanonicalValue(Raw raw)
         return raw.value >= raw.range.min && raw.value <= raw.range.max;
 }
 
+// Shared clamping utility.
+template<typename Raw> Raw performParseTimeClamp(Raw raw)
+{
+    static_assert(raw.range.options != CSS::RangeOptions::Default);
+
+    if constexpr (raw.range.options == CSS::RangeOptions::ClampLower) {
+        return { std::max<typename Raw::ValueType>(raw.value, raw.range.min) };
+    } else if constexpr (raw.range.options == CSS::RangeOptions::ClampUpper) {
+        return { std::min<typename Raw::ValueType>(raw.value, raw.range.max) };
+    } else if constexpr (raw.range.options == CSS::RangeOptions::ClampBoth) {
+        return { std::clamp<typename Raw::ValueType>(raw.value, raw.range.min, raw.range.max) };
+    }
```

Comment:
Not sure why these if statements have braces

Reply:
Good point!.